lib/identity/stateTransitions/assetLock/proof/instant/validateInstantAssetLockProofStructureFactory.js

```diff
@@ -54,9 +54,7 @@ function validateInstantAssetLockProofStructureFactory(
     }
 
     if (!skipAssetLockProofSignatureVerification) {
```

Comment:
Remove `skipAssetLockProofSignatureVerification`. We don't need it anymore since we can just do a fallback inside `verifyInstantLock` if needed.